spec.md

```diff
@@ -314,36 +308,36 @@ a trailing `'`{.firrtl}.  The following is an example of a raw string literal:
 'world'
 ```
 
-## String-encoded Integer Literal
+## Radix-specified Integer Literal
 
-A string-encoded integer literal is a special string literal with one of the
+A radix-specified integer literal is a special integer literal with one of the
 following leading characters to indicate the numerical encoding:
 
-- `b`{.firrtl} -- for representing binary numbers
-- `o`{.firrtl} -- for representing octal numbers
-- `h`{.firrtl} -- for representing hexadecimal numbers
+- `0b`{.firrtl} -- for representing binary numbers
+- `0o`{.firrtl} -- for representing octal numbers
+- `0h`{.firrtl} -- for representing hexadecimal numbers
```

Comment:
It might seem stupid, and maybe it is, but what about `0d`?

Reply:
Can't hurt to allow it. Added this.